SocialFilter/sections/specific.txt

```diff
@@ -63,6 +63,7 @@ moveout.in##.post_share
 celtabettv3.com,turkbet.tv,pasizle5.com,hiltonbettv1.com##.channel-share
 cosmopolitan.de##.sharingList
 turksemedia.nl##.share_box_container
+anbariloche.com.ar##.widget_twitter
```

Comment:
Please also block this:
||platform.twitter.com/js/timeline$domain=anbariloche.com.ar

Either with $domain suffic or third-party suffix.